iOSClient/Viewer/NCViewerMedia/NCViewerMediaPage.swift

```diff
@@ -228,6 +233,22 @@ class NCViewerMediaPage: UIViewController {
         }
     }
 
+    @objc func uploadedFile(_ notification: NSNotification) {
+
+        if let userInfo = notification.userInfo as NSDictionary? {
+            if let ocId = userInfo["ocId"] as? String, let metadata = NCManageDatabase.shared.getMetadataFromOcId(ocId), let errorCode = userInfo["errorCode"] as? Int {
+                if errorCode == 0, let index = metadatas.firstIndex(where: {$0.ocId == metadata.ocId}) {
```

Comment:
Again, please use _one_ `guard` instead of cascading `if`s :)
Makes it clearer and easier to read..